modules/directives/src/main/scala/scala/build/directives/UsingDirectiveValueNumberBounds.scala

```diff
@@ -0,0 +1,22 @@
+package scala.build.preprocessing.directives
+
+case class UsingDirectiveValueNumberBounds(
```

Comment:
It sees that we only have 3 case used:
 - AtMostOne: `UsingDirectiveValueNumberBounds(0,1)`
 - One :  `UsingDirectiveValueNumberBounds(1,1)`
 - AtLeastOne: ` `UsingDirectiveValueNumberBounds(1,MaxInt)`
 
I from the perspective of readablity of error messages I think having a dedicated message for each such case would be more easy to understand then general message, especially if it contains `MaxInt`

Reply:
This is only instantiated once in the `recieveTheRightNumberOfValues`
Breaking it down to multiple values,requires changing the logic therefrom its current clean form, to check for the concrete values of the lower and upper bounds...